style.css

```diff
@@ -338,3 +338,129 @@ ul {
   padding: 0 100px 8px 100px;
   border-radius: 10px;
 }
+
+@media (max-width: 640px) {
+  .main2 {
+    display: none;
+  }
+
+  .card-desktop-img {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 640px) {
+  * {
+    box-sizing: border-box;
+  }
+
+  .heading {
+    display: flex;
+    flex-direction: column;
+    background-image: url(./images/header2x.png);
+    width: 100%;
+    height: auto;
+  }
+
+  .content1 {
+    padding: 30px;
+    margin: 315px 260px;
+  }
+
+  .main {
+    display: none;
+  }
+
+  .card-img {
+    display: none;
+  }
+
+  /* .main2 {
+    list-style-type: none;
+  } */
+
+  .navbar {
+    justify-content: space-between;
+    padding: 48px 130px 48px 130px;
+  }
+
+  .section-card {
+    margin: 112px 215px 36px 215px;
+    width: 1156px;
+    height: 496px;
+    grid-template-columns: 50% 50%;
+  }
+
+  .card-text {
+    padding-top: 40px;
+  }
+
+  .about-me {
+    flex-direction: row;
+    padding: 20px;
+    margin-top: 60px;
+  }
+
+  .content {
+    width: 45%;
+    padding: 150px 125px 220px 120px;
+    margin-left: 70px;
+  }
+
+  #lang {
+    width: 50%;
+    display: flex;
+    padding: 142px 0;
+  }
+
+  .lang-1 {
+    display: flex;
+    flex-direction: column;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    color: rgb(0, 0, 0);
+    justify-content: space-between;
+    margin-bottom: 0;
+    align-items: flex-start;
+  }
+
+  .lang-2 {
+    width: 100%;
+  }
+
+  .lang-list {
+    flex-direction: row;
+  }
+
+  .lang-icons {
+    display: grid;
+    padding-left: 5px;
+    width: 122px;
+    height: 120px;
+    flex-direction: column;
+  }
+
+  .lang-1 > li {
+    width: 100%;
+  }
+
+  #footer {
+    padding: 60px 484px 60px 484px;
+    width: 100%;
+    background-image: url(./images/contact-form2x.png);
+    background-size: 92%;
+    background-position: 2em 12em;
+  }
+
+  #contact-text {
+    line-height: 10px;
+    margin-left: 87px;
+  }
```

Comment:
Kindly make sure that your design sticks as much as possible to the Figma template provided.
- Please center the contact text. :eyes: :eyes: 
![image](https://user-images.githubusercontent.com/98363075/201488179-2d493e43-8a7c-4f0e-af68-29fc6d4195e3.png)

- Kindly make sure that the inputs have the same width that the provided in the Figma template. :+1: :smiley: :smiley: 
Your design
![image](https://user-images.githubusercontent.com/98363075/201488319-c1fabfe6-8691-41fb-a70e-790393fa9e95.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/201488249-7a8d73c2-5120-4b6c-a89c-04d9b80429a6.png)